AstSemantics.md

```diff
@@ -154,6 +151,28 @@ by [available resources](Nondeterminism.md)). In the future, to support
 [>4GiB linear memory](FutureFeatures.md#heaps-bigger-than-4gib), support for
 indices with type `int64` will be added.
 
+### Adjusting memory size and permissions: `mmap`
+
+In the MVP the size of linear memory is fixed: The initial size of linear
+memory will remain unchanged for the life of that WebAssembly module. Later,
```

Comment:
How as in "where" or "what units", or something else?

I assume the current docs imply it is specified in the wasm binary somewhere, "total memory is 4MB".